static/docs/get-started/import-data.md

```diff
@@ -67,9 +67,9 @@ outs:
     persist: false
 ```
 
-The `url` subfield points to the source project, while `rev_lock` lets DVC know
-which Git repository version did the data come from. Note that `dvc update`
-updates the `rev_lock` value.
+The `url` subfield under `repo` points to the source project, while `rev_lock`
+lets DVC know which Git repository version did the data come from. Note that
+`dvc update` updates the `rev_lock` value.
```

Comment:
Do we need to say that `dvc update` updates `rev_lock`? Or anything about `rev_lock` for that matter? Maybe leave it as an implementation detail.

We really don't expect people to edit that manually, while editing `rev` might make some sense.

Reply:
This IS an implementation detail. You're commenting on a paragraph inside a `<details>` section in https://dvc.org/doc/get-started/import-data.

Reply:
> We really don't expect people to edit that manually

As for this: I agree. But I don't think that's the only reason we provide implementation details in our docs.

Reply:
This comment did made me realize we needed to update other docs that explain about `rev` and `rev_lock` though, so thanks! See my latest commit, 89026ced.